app/src/main/res/values/strings.xml

```diff
@@ -643,7 +643,7 @@
     <string name="other">Other</string>
     <string name="rules">Rules</string>
     <string name="pref_compression_method">Compression method</string>
-    <string name="pref_backup_flags_msg">Adding preset for backup options removes the burden of selecting flags every time you backup/restore</string>
+    <string name="pref_backup_flags_msg">Adding a preset for backup options removes the burden of selecting flags every time you backup/restore.</string>
```

Comment:
Preference items do not have a full-stop at the end. If you're going to add a full-stop to one item, you have to add them to all of them.

Reply:
Did for all preference messages